ppr-ui/src/views/dashboard.vue

```diff
@@ -109,14 +180,21 @@ export default class Dashboard extends Vue {
 }
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
```

Comment:
I think scoped is ok here. 
Are you trying to modify some styling the is nested in a Vuetify component?

If `.v-simple-checkbox .theme--light.v-icon` was the issue, you can always keep the components styling scoped and use `::v-deep .v-simple-checkbox .theme--light.v-icon` to modify the specific style.

Reply:
ah okay sounds good I will update that tomorrow morning

Reply:
Actually having it scoped prevents my custom classes in the data-table from rendering. I can see them being applied in the html, but they don't register any of the css. The weird thing is they don't get overwritten, they just don't show that they have any css associated with the class. I tried using the ::v-deep but it doesn't solve it so I am leaving it unscoped for now. I removed the unneeded classes from there so all the defined classes are unique to this component now

Reply:
the latest best practices doc suggests using **module** instead of **scoped** to fully avoid css collisions.